x-pack/plugins/apm/public/components/app/service_map/controls.tsx

```diff
@@ -19,9 +19,9 @@ import { getAnimationOptions, getNodeHeight } from './cytoscape_options';
 import { useAnyOfApmParams } from '../../../hooks/use_apm_params';
 
 const ControlsContainer = euiStyled('div')`
-  left: ${({ theme }) => theme.eui.gutterTypes.gutterMedium};
+  left: ${({ theme }) => theme.eui.euiSize};
```

Comment:
Is euiSize medium? or there's a euiSizeM?

Reply:
`euiSize` is medium: https://github.com/elastic/eui/blob/ff73b1f80dcf7500d57862348c746161a6ec6b9b/src-docs/src/views/theme/_json/eui_theme_light.json#L333

There is no `euiSizeM` in our Sass variables / JS variables exported from our Sass via JSON, but there is in our new JS-first `euiTheme`, used in our CSS-in-JS. Our naming generally strives to be clearer in our new CSS-in-JS code than in our older Sass code!